src/zenodo_client/api.py

```diff
@@ -37,24 +37,34 @@ def ensure_zenodo(key: str, data: Data, paths: Paths, **kwargs) -> requests.Resp
     return Zenodo(**kwargs).ensure(key=key, data=data, paths=paths)
 
 
-def create_zenodo(data: Data, paths: Paths, **kwargs) -> requests.Response:
+def create_zenodo(data: Data, paths: Paths, *, publish: bool = False, **kwargs) -> requests.Response:
     """Create a Zenodo record."""
-    return Zenodo(**kwargs).create(data, paths)
+    return Zenodo(**kwargs).create(data, paths, publish=publish)
 
 
-def update_zenodo(deposition_id: str, paths: Paths, **kwargs) -> requests.Response:
+def update_zenodo(deposition_id: str, paths: Paths, *, publish: bool = True, **kwargs) -> requests.Response:
     """Update a Zenodo record."""
-    return Zenodo(**kwargs).update(deposition_id, paths)
+    return Zenodo(**kwargs).update(deposition_id, paths, publish=publish)
+
+
+def update_metadata_zenodo(deposition_id: str, data: Metadata, *, publish: bool = True, **kwargs) -> requests.Response:
+    """Update the metadata of a Zenodo record."""
+    return Zenodo(**kwargs).update_metadata(deposition_id, data, publish=publish)
+
+
+def publish_zenodo(deposition_id: str, *, sleep: bool = True, **kwargs) -> requests.Response:
+    """Publish a Zenodo record."""
+    return Zenodo(**kwargs).publish(deposition_id, sleep=sleep)
 
 
 def download_zenodo(deposition_id: str, name: str, force: bool = False, **kwargs) -> Path:
     """Download a Zenodo record."""
-    return Zenodo(**kwargs).download(deposition_id, name=name, force=force)
+    return Zenodo(**kwargs).download(record_id=record_id, name=name, force=force)
 
 
-def download_zenodo_latest(deposition_id: str, path: str, force: bool = False, **kwargs) -> Path:
```

Comment:
please don't make cosmetic changes like this within a PR that includes other additions and changes

@nuest I also made several reverts before, do you understand why I am asking for this? If not, I can explain further

Reply:
No, I understand - I respect you for keeping your codebase and commit history clean like that, something to aspire to.